utils/build.config

```diff
@@ -8,6 +8,7 @@ PMDK = 1.8
 ISAL = v2.26.0
 SPDK = v19.04.1
 FUSE = fuse-3.6.1
+FIO = fio-3.20
```

Comment:
Why is this needed now and wasn't when 1.0 was created?

Reply:
It was definitely needed as proved by the fact that it doesn't build now.  The fact that it wasn't there was a bug.

Note that fio isn't used in master anyway so this is just a workaround for anyone that checks out 1.0

Reply:
Yeah, I knew about the removal in master.

I'm still puzzled how any of, say, the 1.0.0 RCs built if this was needed.  Maybe I will find a minute to investigate.

Reply:
So it seems that 1.0.0rc2 (for example) did actually checkout and build fio, even though it was missing from `build.config`.  It must have just build the default checkout branch (probably master) after the clone and ended up building:

```
FIO_VERSION = fio-3.20-38-g14060
```

Reply:
yeah, I think there has been a regression in fio master since we built it.   It failed my build before I added the 2nd commit